sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -75,13 +94,13 @@ class ParquetFilters(
   }
 
   /**
-   * Holds a single field information stored in the underlying parquet file.
+   * Holds a single primitive field information stored in the underlying parquet file.
    *
-   * @param fieldName field name in parquet file
+   * @param fieldNames field names in parquet file
```

Comment:
I think It still indicates a single field name? Though it could contain multiple identifiers.

Reply:
Yes, still single field with multi parts. Let me write a better comment about it.